tests/gha-adjust-to-use-prebuilt-components.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2024 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+this_script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+repo_root_dir="$(cd "${this_script_dir}/../" && pwd)"
```

Comment:
We could combine these with
```
repo_root_dir="$(cd "$(dirname "${BASH_SOURCE[0]}/../")" && pwd)"
```
I'm not sure it's a good idea though as it might be less clear?

Reply:
I will leave as it is, as it follows what we have elsewhere.